app/javascript/components/tree-view/hierarchical-tree-view.jsx

```diff
@@ -0,0 +1,93 @@
+/* eslint-disable */
+import React from 'react';
+import PropTypes from 'prop-types';
+import Tree, { ActionTypes } from '@manageiq/react-ui-components/dist/wooden-tree';
+
+class HierarchicalTreeView extends React.Component {
```

Comment:
We are trying not to use classes. please change it to functional component with hooks.